### PR TITLE
Added back button to studentGroup component

### PR DESCRIPTION
**Pull request by SteveLorentzen**

### CODE REVIEW

File: src/screens/studentGroup.tsx [hunk 1]
```diff
@@ -126,9 +133,21 @@ const StudentGroup: React.FC = () => {
...
+    history.push('/')
+  }
+
   return (
     <>
       <Box display="flex" flexDirection="column" alignItems="center">
+        <IconButton
+          icon={<ArrowBackIcon />}
+          aria-label="back"
+          position="absolute"
```
Comment (trevorfehrman) [inline src/screens/studentGroup.tsx]: is this absolutely positioned on the page or relative to its parent?

Comment (SteveLorentzen) [reply src/screens/studentGroup.tsx]: I think it's absolute on the page.  Is that bad?

Comment (trevorfehrman) [reply src/screens/studentGroup.tsx]: yeah sounds brittle, what are you trying to accomplish

Comment (SteveLorentzen) [reply src/screens/studentGroup.tsx]: Just to have the back button near the top left of the screen

Comment (trevorfehrman) [reply src/screens/studentGroup.tsx]: yeah just pulled down the code, i'd put the title and the button in a div, set the div to position relative and position the button absolutely relative to that, the way you're doing it here that back button will remain in that position no matter what, we don't really know what we're doing with respect to the layout yet

Comment (SteveLorentzen) [reply src/screens/studentGroup.tsx]: Okay, awesome.  I'll do that.

Comment (SteveLorentzen) [reply src/screens/studentGroup.tsx]: Finally got around to fixing this

**Review by trevorfehrman — APPROVED** [review]

**Pull request merged**